pkg/routeagent/controllers/route/iptables.go

```diff
@@ -95,3 +96,43 @@ func (r *Controller) programIptableRulesForInterClusterTraffic(remoteCidrBlock s
 
 	return nil
 }
+
+func (r *Controller) clearGlobalnetChains() error {
+	if r.globalnetStatus == GN_Status_Not_Verified {
+		localCluster, err := r.submarinerClientSet.SubmarinerV1().Clusters(r.objectNamespace).Get(r.clusterID, metav1.GetOptions{})
+		if err != nil {
+			return fmt.Errorf("error while retrieving the local ClusterInfo: %v", err)
+		}
+
+		if localCluster.Spec.GlobalCIDR != nil && len(localCluster.Spec.GlobalCIDR) > 0 {
+			r.globalnetStatus = GN_Enabled
+		} else {
+			r.globalnetStatus = GN_Disabled
+		}
```

Comment:
So `globalnetStatus` will only be evaluated once. Could the `GlobalCIDR` on the `Cluster` ever change? If so, is it assumed that would entail  a route-agent restart? Either way it doesn't seem like we really need this optimization to chance it since it's executed on local gateway EP change which should be infrequent.

Reply:
AFAICT we do not support modifying globalnetStatus at runtime. IMHO having this would be helpful (as we can avoid querying cluster object) since it will be periodically invoked every 60 secs.